storage/store.go

```diff
@@ -253,8 +260,48 @@ func (si *storeRangeIterator) Reset() {
 	si.index = 0
 }
 
-// StoreConfig contains various parameters of a Store.
-type StoreConfig struct {
+// A Store maintains a map of ranges by start key. A Store corresponds
+// to one physical device.
+type Store struct {
+	*StoreFinder
+
+	Ident          proto.StoreIdent
+	ctx            StoreContext
```

Comment:
Actually, that would be a pain. I'm fine with making these things public, but the problem is that most of them already are public via accessor methods (used in the RangeManager interface). So we couldn't just expose them without causing conflicts at that level. Probably best to just leave this as it is now.